front-end/src/domains/JobInfoItem/JobInfoItem.tsx

```diff
@@ -0,0 +1,134 @@
+import { useHistory } from "react-router-dom";
+import { AxiosError } from "axios";
+import { useBoolean, useFetch } from "../../hooks";
+import { formatDate, addHours } from "../../utils";
+import { Alert, Button, CalendarIcon, Confirm, Text } from "../../components";
+import { StyledJobInfoItem } from "./JobInfoItem.styled";
+import { API_ENDPOINT, PATH } from "../../constants";
+import { JobViewResponse } from "../../types";
+
+interface JobInfoItemProps extends JobViewResponse {
+  refresh: () => Promise<unknown | AxiosError<unknown>>;
+}
+
+const JobInfoItem = ({
+  id: jobId,
+  name: jobName,
+  status: jobStatus,
+  gpuServerName,
+  memberName,
+  refresh,
+}: JobInfoItemProps) => {
+  // TODO: 실제 예상 시간으로 교체
+  // TODO: 서버별 job의 현재 실행 중인 job의 예상 종료 시간 + 나머지 jobs들의 expected time을 전부 더함
+  // TODO: jobs의 순서 (우선순위)는 index인지 확인 필요
+  const startTime = formatDate(new Date());
+  const endTime = formatDate(addHours(new Date(), Math.floor(Math.random() * 100)));
+
+  const history = useHistory();
+
+  const { status, makeRequest, done } = useFetch<void>(`${API_ENDPOINT.LABS(1).JOBS}/${jobId}`, {
+    method: "put",
+  });
+
+  const [isConfirmOpen, openConfirm, closeConfirm] = useBoolean(false);
+
+  const alertMessage: { [key in "succeed" | "failed"]: string } = {
+    succeed: `${jobName}이(가) 취소되었습니다.`,
+    failed: `${jobName} 취소에 실패하였습니다.`,
+  };
+
+  const handleAlertConfirm = async () => {
+    if (status === "succeed") {
+      await refresh();
+    }
+
+    done();
+  };
+
+  return (
+    <>
+      {(status === "succeed" || status === "failed") && (
+        <Alert onConfirm={handleAlertConfirm}>
+          <Text size="md" weight="regular">
+            {alertMessage[status] ?? ""}
+          </Text>
+        </Alert>
+      )}
```

Comment:
이건 그냥 제안인데 아예 이렇게 두개로 나누는건 어떻게 생각하시나요? ㅎㅎ

```suggestion
      {status === "failed" && (
        <Alert onConfirm={handleAlertConfirm}>
          <Text size="md" weight="regular">
            {alertMessage.failed}
          </Text>
        </Alert>
      )}
      
      {status === "succeed"  && (
        <Alert onConfirm={handleAlertConfirm}>
          <Text size="md" weight="regular">
           {alertMessage.succeed}
          </Text>
        </Alert>
      )}
```